bin/run-caliper/sample-network.json

```diff
@@ -0,0 +1,22 @@
+{
+    "caliper": {
+        "blockchain": "ethereum",
+        "command" : {
+            "start": "docker-compose -f network/ethereum/1node-clique/docker-compose.yml up -d && sleep 3",
```

Comment:
you don't need that. This is running a local node. you need to run the sut in gcloud by calling the deloy sut script

Reply:
ah okay, so how we should change it?

Reply:
i will let this part free for now :D